### PR TITLE
Elevator Shafts | Various Changes

### PR DESCRIPTION
**Pull request by ProbablyCarl**
- - -
### Balance:
 - You can no longer craft magnum shells or flechette outside of specialized roles, in bulk. Instead, you can only craft five at a time, for an obscenely expensive cost. Bulk crafting price increased accordingly.
 - Shotgun stripper clips introduced, in line with the above. Currently, with a few exceptions, you can only obtain flechette and magnum, with the rest not truly being in any real loot pool. Aside from one, which isn't even used, unless I'm mistaken.
 - Grenade launcher removed from high tier ballistic pool.

  NCR:
   - Combat Engineer has lost their rocket launcher loadout.
   - Heavy trooper can no longer spawn with a flamethrower.
   - LO can bulk craft magnum and flechette rounds.
   - The Scout Carbine has been given 10% penetration.

  Legion:
   - 'Loud' Prime Decanus replaced with 'Demolition' Prime Decanus. This replaces the Grenade Launcher with C4, and compensates by providing two additional frags.
   - Forgemaster can bulk craft magnum and flechette rounds.

  Enclave:
   - Combat Technician has lost their grenade launcher. For now.

  Raiders:
   - Outsider Highwayman now spawns with a revolver.

  Khan
   - Senior Enforcer has once again been given the Neostead, although this time as a unique.
   - Senior Enforcer has lost trekking by default.
   - Khan Smith can bulk craft magnum and flechette rounds.

  Bighorn
   - Shopkeep can bulk craft magnum and flechette rounds.
- - -
### Map:
 - Legion slave pens should no longer be unroofed.
 - Upper area of the Legion fortress should now be protected from the elements, when in tiles previously occupied by rock.
 - Initial introduction of elevator locations in Bighorn, Vault 113 and Casper's bunker. These are non-functional for now, but you can jump down the elevator shaft for a one way trip. Areas for these additions have been updated thematically.
 - Bighorn received an extra set of stairs to the underground, next to the gate.
   - Poppy's opioid addicted cousin, Skoggy, will now greet visitors, rather than murdering them.
- - -
### Bugfixes:
 - Outsider should no longer spawn nude. Honest mistake. I swear. Probably.
- - -

### CODE REVIEW

**Comment by ProbablyCarl** [comment]
image
image
image
image
image

**Comment by SigholtStarsong** [comment]
Is the Shopkeeper going to be able to produce those bunk packages of shells as well?

**Comment by ProbablyCarl** [comment]
> Is the Shopkeeper going to be able to produce those bunk packages of shells as well?

Added.

File: code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm [hunk 1]
```diff
@@ -214,19 +214,48 @@
 	name = "12g flechette box"
 	result = /obj/item/ammo_box/shotgun/flechette
 	reqs = list(/obj/item/stack/crafting/metalparts = 1,
-	/obj/item/stack/sheet/prewar = 2,
+	/obj/item/stack/sheet/prewar = 50,//Yes, really.
```
Comment (Spc-Dragonfruits) [inline code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm:217]: I suggest replacing the pre-war metal requirement with either 2 or 3 polymers (alien alloy), or a pre-war steel ingot

File: code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm [hunk 2]
```diff
@@ -214,19 +214,48 @@
...
 	time = 5
 	category = CAT_WEAPONRY
 	subcategory = CAT_AMMO
+	always_available = FALSE
 
 /datum/crafting_recipe/shotgunammomagnum
 	name = "12g magnum shot box"
 	result = /obj/item/ammo_box/shotgun/magnum
 	reqs = list(/obj/item/stack/crafting/metalparts = 2,
+	/obj/item/stack/sheet/prewar = 30,
```
Comment (Spc-Dragonfruits) [inline code/datums/components/crafting/recipes/recipes_weapon_and_ammo.dm:231]: Same as above

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 3]
```diff
@@ -372,6 +372,13 @@
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/tube_noalt
 	force = 10
 
+/obj/item/gun/ballistic/shotgun/automatic/combat/neostead_noalt/khan
+	name = "Cold Water"
+	desc = "Cold Water, the personalised friend of the Senior Enforcer."
```
Comment (Spc-Dragonfruits) [inline code/modules/projectiles/guns/ballistic/shotgun.dm:377]:
```suggestion
	desc = "A personalized Neostead shotgun belonging to a Senior Enforce of the Great Khans."
```

File: code/modules/jobs/job_types/legion.dm [hunk 4]
```diff
@@ -503,14 +503,12 @@ Discuss balance and documentation changes with Dragonfruits#1913 or forward them
...
-	suit_store = /obj/item/gun/ballistic/revolver/grenadelauncher
+	name = "Demolition Prime Decanus"
 	backpack_contents = list(
-		/obj/item/ammo_box/a40mm = 2,
+		/obj/item/grenade/plastic/c4 = 1,
 		/obj/item/gun/ballistic/revolver/m29 = 1,
 		/obj/item/ammo_box/m44 = 3,
-		/obj/item/grenade/f13/frag = 2,
-		/obj/item/grenade/homemade/coffeepotbomb = 1,
+		/obj/item/grenade/f13/frag = 3,
```
Comment (Spc-Dragonfruits) [inline code/modules/jobs/job_types/legion.dm:511]: I'd also replace these with HEs instead, to focus on the demolition bit

**Review by Spc-Dragonfruits — CHANGES_REQUESTED** [review]

**Pull request merged**